inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -74,19 +76,19 @@ endif()
 
 # install
 
-ie_cpack_add_component(${PYTHON_VERSION})
+ie_cpack_add_component(${CPACK_PREFIX}${PYTHON_VERSION})
```

Comment:
can we define a variable `python_component` and use it later instead of manual combination each time

Reply:
Fixed